Cartfile

```diff
@@ -2,7 +2,6 @@ binary "https://www.mapbox.com/ios-sdk/Mapbox-iOS-SDK.json" ~> 3.6
 github "mapbox/MapboxDirections.swift" ~> 0.10
 github "52inc/Pulley" == 1.4
 github "rs/SDWebImage" ~> 4.1
-github "Project-OSRM/osrm-text-instructions.swift" ~> 0.3.2
```

Comment:
The step table needs this dependency: RouteTableViewCell → VisualInstructionFormatter → RouteStepFormatter → OSRMInstructionFormatter.

Reply:
Oh this was a mistake.